src/subdomains/staking/application/services/staking.service.ts

```diff
@@ -65,6 +66,16 @@ export class StakingService {
     return StakingOutputDtoMapper.entityToDto(await this.authorize.authorize(userId, existingStaking.id));
   }
 
+  async getDepositAddressBalance(address: string): Promise<DepositAddressBalanceOutputDto> {
+    const stakingEntity = await this.repository.findOne({
+      where: { depositAddress: { address: address } },
+      relations: ['depositAddress'],
+    });
```

Comment:
What do you get here, if address is undefined?

Reply:
undefined 👍